shoov/modules/custom/shoov_migrate/csv/message/msg_ci_build.csv

```diff
@@ -4,3 +4,4 @@ msg_ci_build2,ci_build1,"In progress","06/02/2015 15:03",
 msg_ci_build3,ci_build2,Error,"07/02/2015 09:00",Some logs
 msg_ci_build4,ci_build3,Queue,"08/02/2015 17:00",
 msg_ci_build5,ci_build4,Error,"09/02/2015 18:00",
+msg_ci_buildXSS,ci_buildXSS,Error,"10/02/2015 19:00","XSS ATTACK NODE <script>alert('xss - compilation - log');</script>"
```

Comment:
THe alert content seems to be wrong it should have the content type/ bundle e.g. `'xss - msg_ci_build - log'` (here and below)